src/validations/access-checker/access.ts

```diff
@@ -25,9 +25,6 @@ export const access: Validation = {
     const address = externalCalls.ownerAddress(deployment.auditInfo)
     if (deployedBeforeDCLLaunch && externalCalls.isAddressOwnedByDecentraland(address)) return OK
 
-    const type = args.deployment.entity.type
-    const accessChecker = accessCheckers[type]
-
-    return accessChecker.validate(args)
+    return accessCheckers[deployment.entity.type].validate(args)
```

Comment:
What could happen if `deployment.entity.type` is not key of accessCheckers?

Reply:
Added stronger type to accessChecker so it won't compile if there is a type that is not in the record.